### DISCUSSION TITLE
Microsoft Search - Object reference not set to an instance of an object

### DISCUSSION THREAD

**Discussion by mariaarcena** [microsoft-search/pnp-modern-search#1682] [Q&A] [answered]
Hello! So I updated the pnp modern search on our tenant and I wanted to try the Microsoft Search (as the data source) and search only "People" and I get this:
Object reference not set to an instance of an object
![image](https://user-images.githubusercontent.com/17426994/147796102-bd50eeb4-8f84-4385-9c84-b39fd69b167e.png)

Am I missing something? 🤔💭

**Comment by wobba** [comment, marked as answer]
I get this as well and it's due to the people source "crashing" when we pass down what fields to return. Seems the beta endpoint is not too stable for people search quite yet.
